### PR TITLE
HelloJs: added display type, typed promise value return for login, typed return value GetAuthReponse

### PR DESCRIPTION
**Pull request by baywet**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://adodson.com/hello.js/#hellogetauthresponse
https://adodson.com/hello.js/#hellologin
(unfortunately the return values are not fully documented, I might author a PR over there as well if I find time)
- [x] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.  

Changes:
- added display type
- typed promise value return for login
- typed return value GetAuthReponse

### CODE REVIEW

**Comment by baywet** [comment]
@andy-ms this is the first time I'm editing this typedef. Apparently the CI breaks because of some linting rule. I tried to fix it without success. I believe this is because modules and global variable hello are not properly declared somehow but I'm unable to fix it. Would you mind helping/providing guidance please?

**Comment by baywet** [comment]
found out the solution removing the forced export as well as the namespace declaration. Code runs, linting should pass this time (passes locally) however I don't know if it's a "good" thing.

**Comment by baywet** [comment]
sorry this PR is a mess but I finally found out the proper solution looking at lodash type definitions. Good to go :) and now people can use the global var if they want (to map to the documentation), no need to import explicitly anymore.

File: types/hellojs/index.d.ts [hunk 1]
```diff
@@ -146,3 +170,6 @@ declare namespace hello {
         logout?: ((callback: () => void | string) => void) | string;
     }
 }
+declare global {
+    let hello: hello.HelloJSStatic;
```
Comment (andy-ms) [inline types/hellojs/index.d.ts]: This shouldn't be necessary given the `export as namespace` at the top.

Comment (baywet) [reply types/hellojs/index.d.ts]: thanks! just pushed a new version, hopefully this one is the good one

**Comment by baywet** [comment]
Hey @andy-ms authors don't seem to be available anymore to review PR's. Any chance you could merge this one please?

**Pull request merged**